extensions/Bilibili/package.json

```diff
@@ -43,7 +50,8 @@
     }
   ],
   "dependencies": {
-    "@raycast/api": "^1.39.3"
+    "@raycast/api": "^1.39.3",
```

Comment:
Do you mind updating to the latest Raycast API version?

Reply:
I do not mind.